### PR TITLE
Add a `recursive` option to the git command.

### PR DESCRIPTION
**Pull request by intellectronica**
Make it possible to clone without submodules by setting recursive to no. Default is yes, so no change is needed
in existing plays.

### CODE REVIEW

File: library/source_control/git [hunk 1]
```diff
@@ -384,7 +398,8 @@ def main():
         if module.check_mode:
             remote_head = get_remote_head(git_path, module, dest, version, repo)
             module.exit_json(changed=True, before=before, after=remote_head)
-        clone(git_path, module, repo, dest, remote, depth, version, bare, reference)
+        clone(git_path, module, repo, dest, remote, depth, version, bare,
+              reference, recursive)
     elif not update:
         # Just return having found a repo already in the dest path
         # this does no checking that the repo is the actual repo
```
Comment (iritmalka) [inline library/source_control/git:405]: :+1:

Comment (idanrounds) [inline library/source_control/git:405]: :+1:

**Comment by avivl** [comment]
:thumbsup:

**Comment by mpdehaan** [comment]
version_added should say "1.5" as this won't be in a dot release.

**Comment by intellectronica** [comment]
@mpdehaan changed version_added to 1.5.

**Comment by mpdehaan** [comment]
Merged in, thank you!  As we were a bit slow getting to this -- sorry, there has been a huge backlog -- the version_added is marked as 1.6.

**Pull request merged**